### ISSUE TITLE
Renaming the dafault branch

### ISSUE THREAD

**Issue by qinkunbao** [apache/incubator-teaclave#647]
To follow the inclusive language best practice, we should rename the default branch to main.

URL:
https://github.com/github/renaming

**Comment by mssun** [comment]
Yes, to do this, we need the help of the INFRA team. Specifically, we can submit an issue to the INFRA team [1].

- https://issues.apache.org/jira/projects/INFRA/issues
